include/slang/numeric/ConstantValue.h

```diff
@@ -314,6 +314,7 @@ struct SLANG_EXPORT ConstantRange {
     int32_t right = 0;
 
     ConstantRange() = default;
+    ConstantRange(std::pair<int32_t, int32_t> range) : left(range.second), right(range.first) {}
```

Comment:
Hmm, it's pretty confusing to swap the left and right like this. It seems like the following lines should be equivalent yeah?
```
ConstantRange(1, 2)
ConstantRange(std::pair(1, 2))
```

Reply:
I was thinking of left and right as being syntactical, corresponding to the way ranges are written in SV.
Although the class does support that, I can see it's more straightforward to match the other constructor.